app/src/main/res/values/strings.xml

```diff
@@ -647,6 +647,7 @@
     <string name="map_auto_rotated">Map rotation adjusted with calibration</string>
     <string name="pref_backtrack_interval" translatable="false">pref_backtrack_interval</string>
     <string name="importing_map">Importing map</string>
+    <string name="importing_image">Importing image</string>
```

Comment:
Can this text be changed to "Scanning" (and the resource renamed)?

Reply:
Fixed